engine/apps/api/tests/test_alert_receive_channel.py

```diff
@@ -1850,3 +1850,185 @@ def test_alert_receive_channel_webhooks_delete(
+    make_alert_receive_channel,
+    make_alert_receive_channel_connection,
+    make_user_auth_headers,
+):
+    organization, user, token = make_organization_and_user_with_plugin_token()
+    source_alert_receive_channel = make_alert_receive_channel(organization)
+    connected_alert_receive_channel = make_alert_receive_channel(organization)
+    make_alert_receive_channel_connection(source_alert_receive_channel, connected_alert_receive_channel)
+
+    # get integrations connected to source integration
+    client = APIClient()
+    url = reverse(
+        "api-internal:alert_receive_channel-connected-alert-receive-channels-get",
+        kwargs={"pk": source_alert_receive_channel.public_primary_key},
+    )
+    response = client.get(url, **make_user_auth_headers(user, token))
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json() == {
+        "source_alert_receive_channels": [],
+        "connected_alert_receive_channels": [
+            {
+                "alert_receive_channel": {
+                    "id": connected_alert_receive_channel.public_primary_key,
+                    "integration": connected_alert_receive_channel.integration,
+                    "verbal_name": connected_alert_receive_channel.verbal_name,
+                    "deleted": False,
+                },
+                "backsync": False,
+            },
+        ],
+    }
+
+    # get source integrations for particular integration
+    url = reverse(
+        "api-internal:alert_receive_channel-connected-alert-receive-channels-get",
+        kwargs={"pk": connected_alert_receive_channel.public_primary_key},
+    )
+    response = client.get(url, format="json", **make_user_auth_headers(user, token))
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json() == {
+        "source_alert_receive_channels": [
+            {
+                "alert_receive_channel": {
+                    "id": source_alert_receive_channel.public_primary_key,
+                    "integration": source_alert_receive_channel.integration,
+                    "verbal_name": source_alert_receive_channel.verbal_name,
+                    "deleted": False,
+                },
+                "backsync": False,
+            },
+        ],
+        "connected_alert_receive_channels": [],
+    }
+
+
+@pytest.mark.django_db
+def test_connected_alert_receive_channels_post(
+    make_organization_and_user_with_plugin_token,
+    make_alert_receive_channel,
+    make_user_auth_headers,
+):
+    organization, user, token = make_organization_and_user_with_plugin_token()
+    source_alert_receive_channel = make_alert_receive_channel(organization)
+    alert_receive_channel_to_connect_1 = make_alert_receive_channel(organization)
+    alert_receive_channel_to_connect_2 = make_alert_receive_channel(organization)
+
+    client = APIClient()
+    url = reverse(
+        "api-internal:alert_receive_channel-connected-alert-receive-channels-get",
```

Comment:
s/-get/-post? (is the reverse for post also available? I see it's the same endpoint, but that could change in the future?)

Reply:
Unfortunately "post" is not reversible. Getting `django.urls.exceptions.NoReverseMatch` when trying to do so.